docs/beman-standard.md

```diff
@@ -0,0 +1,170 @@
+
+1. **[CORE.QUALITY] Highest quality**. Standards track libraries impact
+   countless engineers and, consequently, should be of the highest quality.
+2. **[CORE.PRODUCTION_READY] Production-ready**. Production feedback
+   necessitates reliable, well-documented software.
+3. **[CORE.INDUSTRY_STANDARD] Industry standard technology**. Where there's
+   industry consensus on best practices, we should take advantage. Innovation in
+   tooling and style is not our purpose.
+4. **[CORE.INCLUSIVE] Welcoming and inclusive community**. Broad, useful
+   feedback requires an unobstructed path for using, reviewing, and
+   contributing to Beman libraries. This principle encompasses ergonomics,
+   cross-industry participation, and cultural accommodation.
+
+### Changing this document
+
+This is a living document that adapts to evolving best practices and community
+needs. To make changes:
+
+1. Create a [discourse topic](https://discourse.boost.org) detailing the change
+   and how it aligns with the core principles.
+2. After some community discussion, create a PR with the actual change on
+   [GitHub](https://github.com/beman-project/beman) with a *leads question*
+   label. The PR should also link to the discourse topic.
+3. Continue discussions on the PR and discourse topic.
+4. Await a leads a decision based on the community feedback.
+
+### Conventions
+
+This standard consists of entries that include an all-caps, dot-separated
+identifier for referencing.
+
+With the exception of the core principles, these entries are either rules or
+recommendations.
+
+* **Rules** must be followed in order to conform to this standard. These entries
+  are prefixed by "RULE:".
+* **Recommendations** should be followed in general, but specific circumstances
+  may make this a less-than-ideal choice. Libraries not following a specific
+  recommendation can still conform to this standard. These entries are prefixed
+  by "RECOMMENDATION:".
+
+## License
+
+**[LICENSE.APPROVED]** REQUIREMENT: All Beman libraries must be licensed
+under an approved license. These are:
+
+1. [Apache License v2.0 with LLVM Exceptions](https://llvm.org/LICENSE.txt)
+2. [Boost Software License 1.0](https://www.boost.org/LICENSE_1_0.txt)
+3. [The MIT License](https://opensource.org/license/mit)
+
+**[LICENSE.BOOST]** RECOMMENDATION: All Beman libraries should be licensed under
+the [Apache License v2.0 with LLVM Exceptions](https://llvm.org/LICENSE.txt).
+
+**[LICENSE.CRITERIA]** REQUIREMENT: All approved licenses must meet the
+following requirements:
+
+1. Simple to read and understand.
+2. Permission without fee to copy, use and modify the software for any
+   use (commercial and non-commercial).
+3. Requires that the license appears on all copies of the software source code.
+4. Must not require that the license appears with executables or other binary
+   uses of the library.
+5. Must not require that the source code be available for execution or other
+   binary uses of the library.
```

Comment:
Do we need to have anything here about liability?

Reply:
Do you have any wording in mind?

Reply:
Actually, I think we do not need a policy here.